### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by parulgupta2001**

### CODE REVIEW

File: docs/modal.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                            &lt;p&gt;Some text in the Modal...&lt;/p&gt;
+                        &lt;/div&gt;
+                    &lt;/div&gt;    
+                </code>
+            </pre>
+
+            <h3>Javascript</h3>
+            <pre>
+                <code class="language-javascript">
+                    var modal = document.querySelector(".modal");
```
Comment (Muzammil-khan710) [inline docs/modal.html]: Use es6 variable naming
Let or const

File: docs/docs.js/modal.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+var modal = document.querySelector(".modal");
```
Comment (Muzammil-khan710) [inline docs/docs.js/modal.js]: Use let and const here ?

**Review by Muzammil-khan710 — COMMENTED** [review]
Code is clean and readable
Added comments and update your code

**Review by Muzammil-khan710 — APPROVED** [review]
UI is clean 
Code is well structured and good naming of classes are used
Good to merge!

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Pull request merged**